style/main.css

```diff
@@ -194,6 +194,30 @@ form[role="dialog"][data-type="confirm"].lcdui-alert section {
   left: calc(50% - 17px);
 }
 
+.background-screen {
+  z-index: 2;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: auto;
+  bottom: auto;
+  background-color: white;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-image: url("../img/icon-128.png");
```

Comment:
On a Flame running FxOS 2.0, this is what I see:
![index](https://cloud.githubusercontent.com/assets/1616846/5990549/5a24ab6c-a9b6-11e4-813d-1196dd687ac2.png)